File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/utils/AnimationBuilder.java
package de.markhaehnel.rbtv.rocketbeanstv.utils;

import android.view.animation.AlphaAnimation;
import android.view.animation.Animation;

public class AnimationBuilder {
    public static Animation getFadeOutAnimation() {
        AlphaAnimation anim = new AlphaAnimation(1.0f, 0.0f);
        anim.setDuration(500);
        return anim;
    }

    public static Animation getDelayedFadeOutAnimation() {
        AlphaAnimation anim = new AlphaAnimation(1.0f, 0.0f);
        anim.setStartOffset(8000);
        anim.setDuration(500);
        return anim;
    }
}

File: app/src/main/java/de/markhaehnel/rbtv/rocketbeanstv/utils/AnimationBuilder.kt
package de.markhaehnel.rbtv.rocketbeanstv.utils

import android.view.animation.AlphaAnimation
import android.view.animation.Animation

object AnimationBuilder {
    val fadeOutAnimation: Animation
        get() {
            val anim = AlphaAnimation(1.0f, 0.0f)
            anim.duration = 500
            return anim
        }

    val delayedFadeOutAnimation: Animation
        get() {
            val anim = AlphaAnimation(1.0f, 0.0f)
            anim.startOffset = 8000
            anim.duration = 500
            return anim
        }
}
